apps/full-site-editing/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/full-site-editing",
-	"version": "0.23.0",
```

Comment:
I think the docs are referencing an example PR where this hasn't been updated, so that's why people are missing it. :)